crates/pop-parachains/src/up/mod.rs

```diff
@@ -454,6 +471,12 @@ impl NetworkConfiguration {
 						}
 					}
 				}
+				// Resolve individual collator command to binary
+				if let Some(collator) = table.get_mut("collator").and_then(|p| p.as_table_mut()) {
```

Comment:
No test coverage it seems. Please check that all code that is being added is covered by tests.

Reply:
Added the coverage for this change in the existing test `configure_works`